crates/atuin-client/src/settings.rs

```diff
@@ -788,10 +834,6 @@ impl Settings {
             .set_default("daemon.socket_path", socket_path.to_str())?
             .set_default("daemon.systemd_socket", false)?
             .set_default("daemon.tcp_port", 8889)?
-            .set_default(
```

Comment:
I am not sure about this change.
I noticed duplicate defaults for many parameters here and didn't understand the purpose.